__tests__/integration.test.js

```diff
@@ -135,4 +135,62 @@ describe("/api/articles/:article_id", () => {
     })
   })
 
-});
\ No newline at end of file
+});
+
+
+// api/articles/:article_id/comments
+
+describe("/api/articles/:article_id/comments", () => {
+
+  test.only("GET: Requests to this endpoint should respond with an array of all the comments with the requested article_id, each with the following properties; comment_id, author, body, votes, created_at, article_id", () => {
```

Comment:
Very important to make sure you remove .onlys before committing, we need to know that all the tests are passing before we merge to main.